tests/unit/topic_info_test.py

```
@@ -253,7 +253,7 @@ def test_repr_and_str(topic_info):
     assert "memo='Test topic memo'" in repr_output
     assert "sequence_number=42" in repr_output
     assert "running_hash=0x0102030405060708" in repr_output
-    assert "expiration_time=2021-07-01 00:00:00" in repr_output
+    assert "expiration_time=2021-07-01 00:00:00" in repr_outputgit 
```

Ruff (0.14.11)

Undefined name `repr_outputgit`

(F821)